playground/backend/pkg/errors/errors.go

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
That's grpc related errors. Let's name the file grpcErrors?

Reply:
Done